src/ray/object_manager/object_buffer_pool.cc

```diff
@@ -104,6 +105,12 @@ std::pair<const ObjectBufferPool::ChunkInfo &, ray::Status> ObjectBufferPool::Cr
     const ObjectID &object_id, const rpc::Address &owner_address, uint64_t data_size,
     uint64_t metadata_size, uint64_t chunk_index) {
   std::lock_guard<std::mutex> lock(pool_mutex_);
+  if (!pull_manager_->IsObjectActive(object_id)) {
```

Comment:
This can happen pretty commonly right? I wonder if we want to change this part from object manager to avoid spam messages; 

```
  if (chunk_status.second.ok()) {
    // Avoid handling this chunk if it's already being handled by another process.
    std::memcpy(chunk_info.data, data.data(), chunk_info.buffer_length);
    buffer_pool_.SealChunk(object_id, chunk_index);
  } else {
    num_chunks_received_failed_++;
    RAY_LOG(INFO) << "ReceiveObjectChunk index " << chunk_index << " of object "
                  << object_id << " failed: " << chunk_status.second.message()
                  << ", overall " << num_chunks_received_failed_ << "/"
                  << num_chunks_received_total_ << " failed";
  }
```
Maybe we can add a new `if (chunk_status.IsIOError())`?

Reply:
It shouldn't be happening that often, only when there's possibility of thrashing.